### PR TITLE
Make /blacklist show parse /showblacklist

### PR DESCRIPTION
**Pull request by panpawn**
Apparently, a lot of user's don't know that /showblacklist is a thing, so having this be a thing, and apart of blacklist's help command, should make that better known

### CODE REVIEW

File: commands.js [hunk 1]
```diff
@@ -2007,6 +2007,7 @@ exports.commands = {
 	blacklist: function (target, room, user) {
 		if (!target) return this.parse('/help blacklist');
 		if (!this.canTalk()) return this.errorReply("You cannot do this while unable to talk.");
+		if (target === 'show') return this.parse('/showblacklist');
```
Comment (Zarel) [inline commands.js]: But what if a user is named "show"?

Comment (panpawn) [inline commands.js]: Then that user is pretty lucky

Comment (panpawn) [inline commands.js]: Also:
[23:09] +panpawn: can you uh do me a favor
[23:10] +panpawn: can you check if user "show" is abandoned or not?
[23:11] &Trickster: it is

Comment (Zarel) [inline commands.js]: We can disable the user to prevent it from causing an issue in practice, but it's inconsistent to do something like that.

Comment (panpawn) [inline commands.js]: I was more implying that this user hasn't been on PS in quite some time, so the chance of them just showing up and randomly causing trouble seems to be unlikely to me

Comment (panpawn) [inline commands.js]: Maybe just adding it to the help command would be more ideal?

Comment (Zarel) [inline commands.js]: Maybe there should be a way to group together multiple commands so showing the help for one shows the help for all of them...

Comment (bumbadadabum) [inline commands.js]: Yeah I don't think this is the way to do it. Even if "show" doesn't come online ever, it's still pretty BP to do it this way.

**Comment by panpawn** [comment]
heh, just noticed, since ``/blacklist`` requires a reason, you guys actually didn't have to worry about not being able to blacklist user "show"

**Comment by panpawn** [comment]
well, at least, it does now

**Pull request closed without merging**